Model/SentryLog.php

```diff
@@ -62,6 +62,11 @@ public function send($message, $logLevel, Monolog $monolog, $context = [])
 
         if ($logLevel >= (int) $config['log_level']) {
             $client = (new Raven_Client($config['domain'] ?? null));
+
+            if(!is_null($config['environment'])) {
```

Comment:
This isn't PSR compliant, don't know if anything is but it should. @rakibabu maybe integrate https://styleci.io?

```suggestion
            if (!is_null($config['environment'])) {
```